### PR TITLE
added a swap plugin

### PR DESCRIPTION
**Pull request by Lavenderjiang**
added a swap plugin

### CODE REVIEW

File: SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+public class SortWithSwapsPlugin implements GamePlugin {
+    private int ONE = 1;
+    private int TEN = 10;
+
+    // Describes whether the current move is over
+    public Boolean move = false;
+
+    // An array of the numbers 1-10
+    private int[] arr = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
```
Comment (b0bbbert) [inline SortWithSwapsPlugin.java]: Undescriptive array name

File: SortWithSwapsPlugin.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    // The game framework
+    private GameFramework framework = null;
+
+    // The last number clicked, which is to be swapped with the next number clicked
+    private int last = -1;
+
+    //The number of swaps the user has made
+    static Integer count = null;
```
Comment (b0bbbert) [inline SortWithSwapsPlugin.java]: No need to be and Integer, use int instead
Also access modifier should be private

File: SortWithSwapsPlugin.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            count = count + 1;
+        } else {
+            last = x;
+        }
+
+        this.move = !this.move;
+    }
+
+    public boolean isGameOver() {
+//        System.out.println("here");
```
Comment (b0bbbert) [inline SortWithSwapsPlugin.java:98]: Remove debugging print statement

File: SortWithSwapsPlugin.java [hunk 4]
```diff
@@ -0,0 +1,118 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+
+import java.util.Random;
+
+public class SortWithSwapsPlugin implements GamePlugin {
+    private int ONE = 1;
+    private int TEN = 10;
```
Comment (b0bbbert) [inline SortWithSwapsPlugin.java:10]: Appear to be constants, make private static final

File: SortWithSwapsPlugin.java [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+    // Describes whether the current move is over
+    public Boolean move = false;
```
Comment (b0bbbert) [inline SortWithSwapsPlugin.java:13]: Shouldn't be public

**Review by b0bbbert — COMMENTED** [review]
fix issues

File: SortWithSwapsPlugin.java (hunk 2 above)
Comment (Lavenderjiang) [reply SortWithSwapsPlugin.java]: accepted

**Pull request merged**